### DISCUSSION TITLE
Taskwiki partially working

### DISCUSSION THREAD

**Discussion by ghost** [tbabej/taskwiki#328] [Community support]
I see the filter sections that hide automatically, but pressing `<CR>` or trying to run :TaskWikiInfo doesn't do anything. 
I don't see any :Task* commands listed, but I do see :Vimwiki* commands

There's no errors on startup, and `task` returns nothing

I'm using Pathogen and followed the guide:
Vimwiki - dev
Taskwiki - master 
Tasklib - 2.3.0
nvim - 0.3.0

**Comment by tbabej** [comment]
Hi @mrsleeeeve,

on `nvim` you have to ensure that you have python3 support available. See `:help python3` for more help, but TL;DR you can test using:

`:python3 print('hello world')`

**Reply by ghost** [reply]
Hi!

looks like python is all good, nvim printed "hello world"

is there anything else I can check/provide?

**Comment by tbabej** [comment]
Try importing tasklib through vim

:python3 import tasklib; print(tasklib.__file__)

-------- Original Message --------

**Reply by ghost** [reply]
it returned

`/usr/local/lib/python3.6/site-packages/tasklib/__init__.py`
